apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.html.ts

```diff
@@ -141,28 +158,29 @@ function EditConfiguration({
   infoCourseInstancePath: string;
 }) {
   if (!hasCoursePermissionView && !hasCoursePermissionEdit) {
-    return 'infoCourseInstance.json';
+    return '';
   }
 
-  if (hasCoursePermissionEdit && !exampleCourse) {
+  if (!hasCoursePermissionEdit || exampleCourse) {
     return html`
-      <a href="${urlPrefix}/${navPage}/file_view/${infoCourseInstancePath}">
-        infoCourseInstance.json
-      </a>
-      <a
-        class="btn btn-xs btn-secondary mx-2"
-        role="button"
-        href="${urlPrefix}/${navPage}/file_edit/${infoCourseInstancePath}"
-      >
-        <i class="fa fa-edit"></i>
-        <span>Edit</span>
-      </a>
+      <p class="mb-0">
+        <a href="${urlPrefix}/${navPage}/file_view/${infoCourseInstancePath}">
+          View course instance configuration
+        </a>
+        in <code>infoCourseInstance.json</code>
+      </p>
     `;
   } else {
     return html`
-      <a href="${urlPrefix}/${navPage}/file_view/${infoCourseInstancePath}">
-        infoCourseInstance.json
-      </a>
+      <p class="mb-0">
+        <a
+          data-testid="edit-course-instance-configuration-link"
+          href="${urlPrefix}/${navPage}/file_edit/${infoCourseInstancePath}"
+        >
+          Edit course instance configuration
+        </a>
+        in <code>infoCourseInstance.json</code>
```

Comment:
Nit: it's subtle, but the underlining extends into the space after the link:

<img width="343" alt="Screenshot 2025-02-05 at 14 10 50" src="https://github.com/user-attachments/assets/11c63cae-8751-4a94-b27a-0b7ba9b61afa" />

You can resolve that by ensuring there isn't a space between the text and the closing `</a>`. Here's a start, you'll have to let Prettier do its thing with this. This is the one case where my usual desire to have tags and text on separate lines doesn't work out.

```suggestion
        <a
          data-testid="edit-course-instance-configuration-link"
          href="${urlPrefix}/${navPage}/file_edit/${infoCourseInstancePath}"
        >Edit course instance configuration</a>
        in <code>infoCourseInstance.json</code>
```

I've noticed this is an issue on some of the editor pages too, might be nice to resolve it?